source/assets/css/visual-design/_theme.scss

```diff
@@ -328,7 +328,7 @@ html {
 }
 
 body {
-  background: transparent;
+  background: white;
```

Comment:
This is going to cover up the background texture on the `html` element, isn't it?

Reply:
Oops! Thank you for catching this. This should be removed; I was doing it to make it easier to auto-run contrast checks. 🙃

Reply:
Let's keep the transparent there in case someone has user styles defined.

Reply:
Yep! Updated it in the latest commit. 💙